packages/material-renderers/src/util/debounce.ts

```diff
@@ -25,15 +25,27 @@
 import debounce from 'lodash/debounce';
 import { useState, useCallback, useEffect } from 'react';
 
-const eventToValue = (ev: any) => ev.target.value;
-export const useDebouncedChange = (
-  handleChange: (path: string, value: any) => void,
-  defaultValue: any,
-  data: any,
-  path: string,
-  eventToValueFunction: (ev: any) => any = eventToValue,
-  timeout = 300
-): [any, React.ChangeEventHandler, () => void] => {
+const defaultEventToValue = (ev: any) => ev.target.value;
+interface DebouncedChangeParams {
+  handleChange: (path: string, value: any) => void;
+  data: any;
+  path: string;
+  eventToValue?: (ev: any) => any;
+  defaultValue?: any;
+  flushOnBlur?: boolean;
+  focused?: boolean;
+  timeout?: number;
+}
+export const useDebouncedChange = ({
```

Comment:
I know it's a lot of parameters already, but I would prefer to keep the old function and just add more optional parameters to it. The reason is that this change needlessly breaks backward compatibility for adopters.

Reply:
Sure thing. I'll fix this asap!